operator/src/test/java/oracle/kubernetes/operator/StartupControlTest.java

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
remove "All rights reserved"

Reply:
They all say that. Is this a change?

Reply:
@markxnelson The operator runtime build failed if I remove "All rights reserved" part with the following warning.
[WARNING] src/main/java/oracle/kubernetes/operator/StartupControl.java:[1] (header) RegexpHeader: Line does not match expected header line of '^// Copyright \(c\) (\d\d\d\d, )+Oracle Corporation and\/or its affiliates\.  All rights reserved\.$'.

Reply:
left unchanged.